src/components/SlateEditor/plugins/blockPicker/actions.tsx

```diff
@@ -62,7 +62,6 @@ export interface Action {
   data: ActionData;
   icon: JSX.Element;
   bookmark?: string;
-  requiredScope?: string;
```

Comment:
Tenker det kan være lurt å ha med denne. Det er fleire gonger vi har hatt behov for å gi tilgang til en funksjon kun for enkelte roller.

Reply:
La det tilbake. Forventer at du bruker det nå!!